editor/test/integration/collection_test.clj

```diff
@@ -250,3 +248,144 @@
                    (test-util/with-prop [inst-id :id "props"]
                      (is (g/error? (test-util/prop-error inst-id :id)))
                      (is (build-error? coll-id)))))))))
+
+(defn- vector3-pb
+  ^DdfMath$Vector3 [^double x ^double y ^double z]
+  (-> (DdfMath$Vector3/newBuilder)
+      (.setX x)
+      (.setY y)
+      (.setZ z)
+      (.build)))
+
+(deftest component-ddf-scale
```

Comment:
Because of the "fake defaults" behavior of `editor.protobuf/read-text` (that I'm too scared to change in this already substantial PR), this test is written in terms of the generated protobuf Java classes. This enables us to verify exactly which values get written to the resulting pb-text and binary files.